pkg/controller/resourcequota/resource_quota_controller_patch.go

```diff
@@ -1,86 +1,92 @@
 /*
- Copyright 2022 The KCP Authors.
+Copyright 2022 The KCP Authors.
 
- Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Likely not the diff you want ;)

Reply:
Thanks, had some whitespace issues. Will fix.

Reply:
Fixed